content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,256 @@
+
+* Web applications MUST support the HTTPS 1.1 protocol.
+* Application SHOULD support TLS.  (Please refer to the [code TLS Guidance][code TLS Guidance] document for further detailed practices related to TLS.)
+* Application SHOULD use (only) port 443 for TLS.
+* Application SHOULD support and prefer TLS 1.2 or higher.
+* Application SHOULD have a valid, non-expired TLS certificate issued from a [trusted authority](https://github.com/scott-bach/fhir.cerner.com/blob/patch-1/content/authorization/tls-guidance.md#trusted-certificate-authorities).
+* Application SHOULD use an OV or EV certificate, if possible, instead of a DV certificate.
+* Application's web site SHOULD [host a DNS TXT record](#setting-up-your-domains-dns-txt-record) containing the Cerner-issued client identifier for the application (also referred to as a "system account ID" in CernerCentral or a "client identifier" in the [OAuth 2 specification][OAuth 2 specification] ).
+* Application's name SHOULD ideally align with the name of the website hosting it and/or the legal entity operating it.
+* Mobile/Native applications SHOULD (as suggested by [RFC-8252][RFC 8252] ) use [claimed "https" Scheme URI redirection](https://datatracker.ietf.org/doc/html/rfc8252#section-7.2) instead of private-use URI scheme redirection.
+
+
+Adhering to all the above requirements ensures that Cerner can convey the most meaningful, descriptive information about your application to end users. Failure to follow recommended (SHOULD) requirements will result in users seeing less information about your application. When Cerner is unable to determine information about your application because one or more of the requirements above have not been implemented, Cerner will display "unknown" when describing your application to users.  See the "Information Displayed to Users" section below for more details.
+
+If you'd like to better understand why Cerner has these technical requirements, please see the "*Information Displayed to Users*" and "*FAQ*" sections below.
+
+NOTE:  Please consider the [code Validation Program][code Validation Program] if you would like Cerner to perform a deeper security assessment of your application.
+
+### Setting Up Your Domain's DNS TXT Record ###
+
+To follow the security best practices, you will need to add a DNS TXT record to your appliction's website containing the Cerner-issued client identifier for the application (also referred to as a "system account ID" in CernerCentral or a "client identifier" in the [OAuth 2 specification][OAuth 2 specification] ).
+
+**Please consult your ISP's documentation for the steps to set up a DNS TXT record at your application's domain.**
+
+The DNS TXT record of your application must match the following key-value format:
+
+"cerner-client-id={value}"
```

Comment:
Is `{}` around the value required?

Look at cerner.com DNS TXT entry, 

```
nslookup -type=TXT cerner.com
Server:		149.45.239.1
Address:	149.45.239.1#53

cerner.com	text = "pardot797603=c6f53f123dd495dcc490d60b553b8065031294131712cc784ab1b149c693d11f"
cerner.com	text = "Cerner Corporation"
cerner.com	text = "teamviewer-sso-verification=961acf23230a4c57b9af88ca55f4b099"
cerner.com	text = "apple-domain-verification=vqIowYBzbwHqtwKQ"
cerner.com	text = "eUpH3u6VVF4+2GpgVvQnNj58uZe1GbboR3F8Jll8BbIu8Tj97o/yvBzZSvK6jeciKUKzORbADEEAVbmRa0QKXQ=="
```

It's just "key=value"

Reply:
Is it confusing? It is a pretty standard way to say:  {this is configurable or put actual value here}

Reply:
To me, I thought we need the {} too. I think having an example would clear the confusion.

Reply:
Something like this?
```
Example:
cerner-client-id = "bxdcd0ff-5der-4c15-b2tf-r28b105aa845"
```

Reply:
@scott-bach 

I think it's `"cerner-client-id=bxdcd0ff-5der-4c15-b2tf-r28b105aa845"` but I'll let @mbeerman  or others confirm.

Reply:
I verified that an example would be cerner-client-id=bxdcd0ff-5der-4c15-b2tf-r28b105aa845 without the quotes. So I added `Example:  
cerner-client-id=bxdcd0ff-5der-4c15-b2tf-r28b105aa845`